src/js/experimental/media-playback-rate-listbox.js

```diff
@@ -0,0 +1,73 @@
+import MediaChromeListbox from './media-chrome-listbox.js';
+import { window, document } from '../utils/server-safe-globals.js';
+import { MediaUIAttributes, MediaUIEvents } from '../constants.js';
+
+const slotTemplate = document.createElement('template');
+slotTemplate.innerHTML = `
+  <style>
+    media-chrome-listitem {
+      white-space: var(--media-captions-listbox-white-space, nowrap);
+    }
+
+  </style>
+`;
+
+class MediaPlaybackrateListbox extends MediaChromeListbox {
+  static get observedAttributes() {
+    return [
+      ...super.observedAttributes,
+      'aria-multiselectable',
+      MediaUIAttributes.MEDIA_PLAYBACK_RATE
+    ];
+  }
+
+  constructor() {
+    super({slotTemplate});
+  }
+
+  attributeChangedCallback(attrName, oldValue, newValue) {
+    if (attrName === MediaUIAttributes.MEDIA_PLAYBACK_RATE && oldValue !== newValue) {
+      this.value = newValue;
+    } else if (attrName === 'aria-multiselectable') {
+      // diallow aria-multiselectable
+      this.removeAttribute('aria-multiselectable');
+      console.warn("Captions List doesn't currently support multiple selections. You can enable multiple items via the media.textTrack API.");
```

Comment:
could this comment be removed for playbackrate? that's a string of 0.12kB 😛

Reply:
Oops, this can definitely be trimmed, at the very least.